src/AzDoBridge/Controllers/AlexaController.cs

```diff
@@ -60,11 +62,29 @@ public static class AlexaController
                 log.LogError($"Failed to read skill from request message: {ex.Message}", ex);
                 throw;
             }
-
+            
             switch (skillRequest.Request.Type)
             {
                 case RequestType.LaunchRequest:
-                    return ResponseBuilder.Ask($"Ok!", null);
+                {
+                     //Authenticate and maintain User through session 
```

Comment:
Please do not directory implement intents or launch requests into the controller.
The controller is only a middleware.

Please use a factory like `AzDoBridgeActionFactory`

Maybe we could use the same factory (if names are unique) or we need two factories (one for every Request Type)